File: SDKs/amazonaws.com/accessanalyzer/2019-11-01/python/src/sdk/models/shared/substring.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class Substring:
    r"""Substring
    A reference to a substring of a literal string in a JSON document.
    """
    
    length: int = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('length') }})
    start: int = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('start') }})
    

File: SDKs/amazonaws.com/accessanalyzer/2019-11-01/java/lib/src/main/java/openapisdk/models/shared/Substring.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Substring
 * A reference to a substring of a literal string in a JSON document.
**/
public class Substring {
    @JsonProperty("length")
    public Long length;
    public Substring withLength(Long length) {
        this.length = length;
        return this;
    }
    @JsonProperty("start")
    public Long start;
    public Substring withStart(Long start) {
        this.start = start;
        return this;
    }
}